### ISSUE TITLE
Few tests are failing in Fedora

### ISSUE THREAD

**Issue by ignatenkobrain** [netvl/xml-rs#171]
```rust
failures:

---- sample_1_short stdout ----
thread 'sample_1_short' panicked at '
-------------------
Unexpected event at line 32:
Expected: Characters("Cascading style sheet: © - ҉")
Found:    Characters("Cascading style sheet: © - \u{489}")
-------------------
', tests/event_reader.rs:388:21
note: Run with `RUST_BACKTRACE=1` for a backtrace.

---- sample_1_full stdout ----
thread 'sample_1_full' panicked at '
-------------------
Unexpected event at line 50:
Expected: Characters("\n                Cascading style sheet: © - ҉\n            ")
Found:    Characters("\n                Cascading style sheet: © - \u{489}\n            ")
-------------------
', tests/event_reader.rs:388:21

---- sample_5_short stdout ----
thread 'sample_5_short' panicked at '
-------------------
Unexpected event at line 4:
Expected: Characters("test ©≂̸")
Found:    Characters("test ©≂\u{338}")
-------------------
', tests/event_reader.rs:388:21
```

**Comment by netvl** [comment]
This is fixed in the current master. Sorry for the delay!

**Comment by ignatenkobrain** [comment]
Could you release new version of crate?

**Comment by netvl** [comment]
@ignatenkobrain yes, I will do that very soon, once I clean up the build of the project a bit.

**Comment by netvl** [comment]
Released in 0.8.1
